src/modules/new-request-form/ticket-form-field/TicketFormField.tsx

```diff
@@ -52,23 +41,19 @@ export function TicketFormField({
 
   return (
     <>
-      <input
-        type="hidden"
-        name={ticketFormField.name}
-        value={ticketFormField.value as string}
-      />
-      {ticketForms.length > 1 && (
+      <input type="hidden" name={field.name} value={field.value as string} />
+      {field.options.length > 1 && (
         <GardenField>
-          <Label>{label}</Label>
+          <Label>{field.label}</Label>
           <Combobox isEditable={false} onChange={handleChange} ref={ref}>
-            {ticketForms.map(({ id, url, display_name }) => (
+            {field.options.map((option) => (
               <Option
-                key={id}
-                value={url}
-                label={display_name}
-                isSelected={ticketFormField.value === id}
+                key={option.name}
```

Comment:
It would be better to use the `value` as a key. Even if it is a remote possibility, it is possible to have multiple ticket fields with the same label